.gitignore

```diff
@@ -403,7 +403,8 @@ src/multilspy/language_servers/omnisharp/static/
 src/multilspy/language_servers/rust_analyzer/static/
 src/multilspy/language_servers/typescript_language_server/static/
 src/multilspy/language_servers/dart_language_server/static/
+src/multilspy/language_servers/clangd_language_server/clangd/
```

Comment:
I would recommend preserving the naming convention. Everything downloaded at runtime should go under static/